06-lifecycle_of_a_Patch.Rmd

```diff
@@ -22,6 +22,71 @@ To submit a patch, you need:
 
 Refer to the guidelines given [here](https://www.r-project.org/bugs.html#how-to-submit-patches) for the process to prepare a patch against R-devel.
 
+You should modify the source code on your computer of the latest version of R (`r-devel`). You can retrieve the latest source code of R via:
```

Comment:
I can't add a comment to line 19 as you haven't changed it, but that should link to a section with instructions obtaining the R-devel source code with SVN. Or if we don't have a single link, then we need a little more explanation somewhere, so it is clear that these instructions continue on from the ones in that chapter. (Simply pointing to the chapter is not enough, as installing R-devel from a binary would not be sufficient here).